manifests/cirunner.pp

```diff
@@ -34,52 +34,36 @@
 # Copyright 2015 Tobias Brunner, VSHN AG
 #
 class gitlab::cirunner (
-  $concurrent = undef,
-  $metrics_server = undef,
-  $hiera_default_config_key = 'gitlab_ci_runners_defaults',
-  $hiera_runners_key = 'gitlab_ci_runners',
-  $manage_docker = true,
-  $manage_repo = true,
-  $xz_package_name = 'xz-utils',
-  $package_ensure = installed,
-  $package_name = 'gitlab-runner',
+  Optional[Integer]       $concurrent           = undef,
```

Comment:
datatypes \o/

----

I really love them, but adding them to existing params should be a dedicated PR

Reply:
cirunner.pp is a complete re-write so im not sure a dedicated PR makes sense